src/index.js

```diff
@@ -219,6 +219,12 @@ exports.createFromJSON = function (obj, callback) {
   }
 }
 
+exports.isPeerId = function (peerId) {
+  if (peerId.constructor && peerId.constructor.name) {
+    return peerId.constructor.name === 'PeerId'
```

Comment:
Saw this comment after pushed my change. Seems that Symbols won't work since they are different if different instances are created:

![image](https://cloud.githubusercontent.com/assets/1211152/24357780/a94fe9a8-12f6-11e7-8aef-c3e02ed4ea7d.png)

Making two versions (even patch) of peer-id different.